### ISSUE TITLE
WPID assignment for new pathways is again using old WPIDs from deleted pathways

### ISSUE THREAD

**Issue by khanspers** [wikipathways/wikipathways.org#36]
This was an issue we discovered several years ago and I think there was a small fix in the way WPIDs are assigned for new pathways, basically keeping count of the highest WPID used and then always adding 1 to that when assigning new IDs. Looks like this is not working anymore. 

Looking at the deletion log, you can see that WP4206 was deleted on Jan 23, but was assigned to pathway created on Jan 25. See the top 3 entries in the deletion log, these were all reassigned.

<img width="757" alt="screen shot 2018-01-26 at 11 16 23 am" src="https://user-images.githubusercontent.com/1442197/35456490-88ee4cc0-028a-11e8-9e66-b2e6468e2934.png">

**Comment by khanspers** [comment]
Sorry that was unclear. I meant the top 3 entries in the deletion log corresponds to two WPIDs that were recycled.

**Comment by AlexanderPico** [comment]
We reviewed this bug and noticed that it's restricted to the case where the very latest created pathway is deleted, then that ID is reused upon the next creation. If any pathway is created in the meantime, then it is not reused. 

This is rare and NOT likely to cause confusion since the deleted pathway only existed for a few hours to a few days max (i.e., when another pathway created).  So, they aren't going to share or publish the WPID or anything in that period of time... at least not irreversibly so.

tl;dr = not a bug.
